changes.go

```diff
@@ -48,8 +49,14 @@ func (cs *changeset) add(method string, requires []string, args ...interface{})
 	if requires == nil {
 		requires = make([]string, 0)
 	}
+	// TODO frankban: current Python bundle lib includes this inconsistency;
+	// break compatibility and just use addService?
+	idPrefix := method
```

Comment:
Can we have versioned changes doc, so that bundlechanges and clients know what to expect?